edx_repo_tools/codemods/django3/remove_python2_unicode_compatible.py

```diff
@@ -0,0 +1,42 @@
+"""
+A script to remove the python2_unicode_compatible imports and headers
+"""
+import sys
+import click
+from bowler import Query
+
+
+def remove_node(node, _, __):
+    """
+    Remove the node containing the expression python_2_unicode_compatible
+    """
+    node.remove()
+
+
+def run_removal_query(path):
+    """
+    Run the bowler query on the input file for refactoring.
+    """
+    (
+        Query(path)
+        .select("decorator<'@' name='python_2_unicode_compatible' any>")
+        .modify(remove_node)
+        .select("import_from<'from' module_name=any 'import' 'python_2_unicode_compatible'>")
+        .modify(remove_node)
+        .write()
+    )
+
+
+@click.command()
+@click.option('--path', help='use syntax: --path {path_to_test_file}')
```

Comment:
shouldn't this also accept directory paths? and iterate over python recursively and remove `python_2_unicode_compatible` decorator

Reply:
It works with the directory path too and refactors all files recursively. I'll update the description.